torch_geometric/loader/node_loader.py

```diff
@@ -65,6 +66,8 @@ def __init__(
         input_nodes: InputNodes = None,
         transform: Callable = None,
         filter_per_worker: bool = False,
+        use_cpu_worker_affinity=False,
+        cpu_worker_affinity_cores=None,
```

Comment:
```suggestion
        use_cpu_worker_affinity: bool =False,
        cpu_worker_affinity_cores: Optional[List[int]] =None,
```
Can we also add some description for them in the doc-string?